config/initializers/devise.rb

```diff
@@ -220,7 +220,7 @@
   # Turn scoped views on. Before rendering "sessions/new", it will first check for
   # "users/sessions/new". It's turned off by default because it's slower if you
   # are using only default views.
-  # config.scoped_views = false
+  config.scoped_views = true
```

Comment:
Without setting scoped_views to true, devise won't pickup changes in views/users folder's views(this is needed because I have changed the registration and edit forms in user views)